docs/cristin-proxy-swagger.yaml

```diff
@@ -1937,6 +2052,22 @@ components:
           en: "Summary in an academic manner"
         popularScientificSummary:
           en: "popularScientificSummary popularScientificSummary popularScientificSummary"
+    ProjectPatchExample:
+      value:
+        title: 'Updated Title'
+        alternativeTitles:
+          - 'en': 'Updated Title in english'
+        startDate: '2022-10-09T00:00:00.001Z'
+        endDate: '2030-12-31T00:00:00.001Z'
```

Comment:
Does it now work with 000Z instead of 001Z?

Reply:
Yes,  CustomInstantSerializer is supposed to take care of that -( when the model comes in place....)